client/src/components/Loading/_Loading.scss

```diff
@@ -1,13 +1,16 @@
-.loading {
-
+.absolute {
```

Comment:
These CSS changes are the right idea, but there's two things we should change. First, the class names `absolute` and `static` are generic and risk conflicting with other classes we might use in the future. In general you want to name your CSS so that it's tough to have duplicate names. Second, we'll want to implement BEM naming here. If you're not familiar with it, please review at http://getbem.com/. Using BEM naming, we can come up with some non-generic names that explain these elements and the hierarchy between them: for example, you could use `.loading--absolute`, `.loading--static`, and `.loading__gif`.